_rules/auto-play-audio-has-control-mechanism-4c31df.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: 4c31df
-name: auto-play audio has a control mechanism
+name: '`autoplay` audio has a control mechanism'
```

Comment:
If autoplay is used as an adjective (?) or a verb, there is no reason for the backticks. In these cases, autoplay does not refer to the autoplay attribute.

To avoid confusion, consider rewriting where autoplay does not refer to the attribute.

Suggestion in this case: Autoplay audio -> Audio that plays automatically

This also goes for the other uses of the word autoplay in this rule.